packages/accordions/src/styled/timeline/StyledOppositeContent.ts

```diff
@@ -15,7 +15,6 @@ export const StyledOppositeContent = styled.div.attrs({
   'data-garden-version': PACKAGE_VERSION
 })`
   flex: 1;
-  margin-right: auto;
```

Comment:
☝️ @hzhu do you know what this was intended for? If needed, it would have to be tuned for LTR.

Reply:
I think this might have been accidentally left from previous iterations, and can be safely removed.